Makefile.Common

```diff
@@ -1,11 +1,11 @@
 # In order to ensure make instructions fail if there is command that fails a pipe (ie: `go test ... | tee -a ./test_results.txt`)
```

Comment:
I think we can revert the changes in this file, right? (not against fixing whitespace, just not related)

Reply:
Whoops, unintentional. Good catch.